src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1

```diff
@@ -53,3 +53,36 @@ function Set-AzKeyVaultSecretFromFile {
 }
 
 Export-ModuleMember -Function Set-AzKeyVaultSecretFromFile
+
+<#
+ .Synopsis
+  Sets a secret as a BASE64 format from a file in Azure Key Vault.
+
+ .Description
+  Sets a secret certificate as a BASE64 format from a file as plain text in Azure Key Vault.
```

Comment:
As this can be used for anything, and is not limited to certificates, we might want to change it to something like this: 
```suggestion
  Uploads the content of a file as a Base64 encoded string, as plain text, into an Azure Key Vault secret.
```